.gitignore

```diff
@@ -24,3 +24,4 @@ benchmark_outputs
 .checkstyle
 .mvn/timing.properties
 .editorconfig
+.presto_history
```

Comment:
It would be nice if this didn't have to live in the source tree, but I guess it's no worse than `var` or anything else that gets created when you run the server for development.